flow_screen_components/datatable/force-app/main/default/lwc/datatable/datatable.js

```diff
@@ -263,6 +263,7 @@ export default class Datatable extends LightningElement {
     @track isAllFilter = false;
     @track showClearButton = false;
     @track tableHeightAttribute = 'height:';
+    @track tableBorderStyle = 'border-left: var(--lwc-borderWidthThin,1px) solid var(--lwc-colorBorder,rgb(229, 229, 229))';
```

Comment:
@ericrsmith35 this was added in response to a request of mine to create a visible left border....

Reply:
Thanks Alex.  Can you think of any reason this should be optional or selectable?

Reply:
not so far...i guess we keep our eyes out for a scenario where it doesn't look good. If so, we make it optional

I also just asked him to do the same thing for the tob border and right border 
![image](https://user-images.githubusercontent.com/3140883/152699064-25f22fe4-5f6b-4aae-b23b-d741c610b758.png)